New .NET AI chat app template quickstart (#44988)
dotnet/docs: docs/ai/quickstarts/ai-templates.md

---
title: Quickstart - Create a .NET AI app using the AI app template
description: Create a .NET AI app to chat with custom data using the AI app template extensions and the Microsoft.Extensions.AI libraries
ms.date: 2/21/2025
ms.topic: quickstart
ms.custom: devx-track-dotnet, devx-track-dotnet-ai
author: alexwolfmsft
ms.author: alexwolf
zone_pivot_groups: meai-targets
---

> # Create a .NET AI app to chat with custom data using the AI app template extensions

Reviewer: The template also supports switching from a local vector store to Azure AI Search. How do you think we should add that into this document? That would introduce a second pivot that's independent of the first one for AI Model Provider.

> In this quickstart, you learn how to create a .NET AI app to chat with custom data using the .NET AI app template extensions. These extensions provide additional starter app templates for Visual Studio and the .NET CLI. The templates are designed to streamline the  getting started experience for building AI apps with .NET by handling common setup tasks and configurations for you.

Reviewer: ```suggestion
In this quickstart, you learn how to create a .NET AI app to chat with custom data using the .NET AI app template.  The template is designed to streamline the  getting started experience for building AI apps with .NET by handling common setup tasks and configurations for you.
```
